client/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
```

Comment:
you don't need module for scss files, remove it

Reply:
hm, you use it for custom-button.. If custom-button will be used in many places of your application, you should move folder custom button into folder shared

Reply:
getting used to this angular folder structure, still thinking in React